client/api_object.go

```diff
@@ -582,3 +586,61 @@ func (c *client) CreateFolder(ctx context.Context, bucketName, objectName string
 	txHash, err := c.CreateObject(ctx, bucketName, objectName, reader, opts)
 	return txHash, err
 }
+
+// GetObjectStatus  return the status of object including the uploading progress
+func (c *client) GetObjectStatus(ctx context.Context, bucketName, objectName string) (string, error) {
```

Comment:
It's better to return an enum structure named `UploadProgress` to the user. Using string type for return values severely hampers programmability. 

And `GetObjectStatus` ==> `GetObjectUploadProgress`

Reply:
updated